packages/tools/lib/create-new-component/index.js

```diff
@@ -108,47 +35,93 @@ const getLibraryName = packageName => {
 	return packageName.substr("webcomponents-".length);
 };
 
+// String manipulation
 const camelToKebabCase = string => string.replace(/([a-z])([A-Z])/g, "$1-$2").toLowerCase();
-
-const packageName = getPackageName();
-const library = getLibraryName(packageName);
-
-const consoleArguments = process.argv.slice(2);
-const componentName = consoleArguments[0];
-
-if (!componentName){
-	console.error("Please enter component name.");
-	return;
+const capitalizeFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1);
+
+// Validation of user input
+const isNameValid = name => typeof name === "string" && name.match(/^[a-zA-Z][a-zA-Z0-9_-]*$/);
+
+const generateFiles = (componentName, tagName, library, packageName, isTypeScript) => {
+	const correctComponentName = capitalizeFirstLetter(componentName);
+	const filePaths = {
+		"extension": isTypeScript 
+			? `./src/${correctComponentName}.ts` 
+			: `./src/${correctComponentName}.js`,
+		"css": `./src/themes/${correctComponentName}.css`,
+		"hbs": `./src/${correctComponentName}.hbs`,
+	};
+
+	const FileContentTemplate = isTypeScript 
+		? tsFileContentTemplate(correctComponentName, tagName, library, packageName) 
+		: jsFileContentTemplate(correctComponentName, tagName, library, packageName);
+
+	// The .extension determines if the file is .js or .ts with the help of the isTypeScript check
+	fs.writeFileSync(filePaths.extension, FileContentTemplate, { flag: "wx+" });
+	fs.writeFileSync(filePaths.css, "", { flag: "wx+" });
+	fs.writeFileSync(filePaths.hbs, "<div>Hello World</div>", { flag: "wx+" });
+
+	console.log(isTypeScript 
+		? `Successfully generated ${correctComponentName}.ts` 
+		: `Successfully generated ${correctComponentName}.js`);
+	console.log(`Successfully generated ${correctComponentName}.css`);
+	console.log(`Successfully generated ${correctComponentName}.hbs`);
+
+	// Change the color of the output
+		console.warn('\x1b[33m%s\x1b[0m', `
+	Make sure to import the component in the bundle.common.js file by using:
```

Comment:
Fix the whole message: print the line they must insert in their bundle file, and don't mention `bundle.common.js`